Objects/enumobject.c

```diff
@@ -80,6 +80,45 @@ enum_new_impl(PyTypeObject *type, PyObject *iterable, PyObject *start)
     return (PyObject *)en;
 }
 
+// TODO: Use AC when bpo-43447 is supported
```

Comment:
I would prefer to track these tasks in the bpo rather than directly in the Python source code.